jina/docker/hubio.py

```diff
@@ -178,6 +178,36 @@ def login(self):
             # use default login
             self._client.login(**get_default_login(), registry=self.args.registry)
 
+    def list(self, name: str = None, kind: str = None, type: str = 'pod', keywords: list = ["numeric", "sklearn"]):
+        """Lists executors allowing filters for name, kind, type and keywords """
+        if not is_db_envs_set():
+            self.logger.error('DB environment variables are not set! aborting list command.')
+            return
+
+        if kind:
+            kind_query = {'manifest_info.kind': kind}
+            sub_query = f'{kind_query}'
+        if type:
+            type_query = {'manifest_info.type': type}
+            sub_query = f'{sub_query}, {type_query}'
+        if name:
+            name_query = {'manifest_info.name': name}
+            sub_query = f'{sub_query}, {name_query}'
+        if keywords:
+            keyword_query = {"manifest_info.keywords":  { $all: keywords } }
+            sub_query = f'{keyword_query}, {name_query}'
+
+        _executor_query = '{ $and: [' + f"""{sub_query}""" + ' ] }'
+        with MongoDBHandler(hostname=os.environ['JINA_DB_HOSTNAME'],
+                            username=os.environ['JINA_DB_USERNAME'],
+                            password=os.environ['JINA_DB_PASSWORD'],
+                            database_name=os.environ['JINA_DB_NAME'],
+                            collection_name=os.environ['JINA_DB_COLLECTION']) as db:
+            existing_doc = db.find(query=_executor_query)
```

Comment:
Currently `db.find` does `find_one`. Should we limit ourselves to just 1 result?

Reply:
Fitting for multiple results.